### PR TITLE
Revise address reservation

### PR DESCRIPTION
**Pull request by Georgeto**
see #224

### CODE REVIEW

**Comment by lukasjuhrich** [comment]
Does the alembic revision control work? This would be the first time we had t use it in production.

File: pycroft/lib/net.py [hunk 1]
```diff
@@ -15,9 +16,14 @@ class MacExistsException(Exception):
...
     for subnet in subnets:
-        reserved = subnet.reserved_addresses or 0
+        reserved_bottom = subnet.reserved_addresses_bottom or 0
+        reserved_top = subnet.reserved_addresses_top or 0
         used_ips = frozenset(ip.address for ip in subnet.ips)
-        unreserved = islice(subnet.address.iterhosts(), reserved, None)
+        unreserved = islice(
+            subnet.address.iterhosts(), reserved_bottom,
+            # Stop argument must be None or an integer: 0 <= x <= sys.maxsize.
+            # IPv6 subnets can exceed this boundary on 32 bit python builds.
```
Comment (lukasjuhrich) [inline pycroft/lib/net.py]: Oh, good catch, would never have thought about that. :+1:

**Comment by lukasjuhrich** [comment]
As CI is broken, I'm assuming you ran the relevant unittests locally?

**Comment by Georgeto** [comment]
> Does the alembic revision control work? This would be its first application.
With the fix in 139e3dd, the migration applied successfully on my local database.

> As CI is broken, I'm assuming you ran the relevant unittests locally?
Yes, I did.

**Comment by lukasjuhrich** [comment]
Splendid! Feel free to merge.

**Pull request closed without merging**